code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -184,10 +184,13 @@
 	foodtype = MEAT | SUGAR
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
+	if(CONFIG_GET(number/monkeycap) <= GLOB.monkeys)
+		visible_message("<span class='warning'>[src] gives a single shudder, before drying out.</span>")
```

Comment:
My comment vanished but you still need to indicate to players why this is happening